### ISSUE TITLE
Unable to run ifconfig

### ISSUE THREAD

**Issue by nirmoy** [cloudius-systems/osv#613]
I am getting below error while running ifconfig

scripts/run.py -nv -e "/tools/ifconfig.so" -- (as per osv/README.md)
OSv v0.19-5-g87460f1
eth0: 192.168.122.89
Failed to load object: /tools/ifconfig.so. Powering off.

I am using 3.13.0-46-generic #79-Ubuntu SMP, Ubuntu 14.04 LTS, Latest OSv source from git

With debug
---------------
OSv v0.19-5-g87460f1
4 CPUs detected
Firmware vendor: Bochs
bsd: initializing - done
VFS: mounting ramfs at /
VFS: mounting devfs at /dev
RAM disk at 0x0xffff8000027b5040 (4096K bytes)
net: initializing - done
vga: Add VGA device instance
eth0: ethernet address: 52:54:00:12:34:56
virtio-blk: Add blk device instances 0 as vblk0, devsize=10737418240
random: virtio-rng registered as a source.
random: intel drng, rdrand registered as a source.
random: <Software, Yarrow> initialized
VFS: unmounting /dev
VFS: mounting zfs at /zfs
zfs: mounting osv/zfs from device /dev/vblk0.1
random: device unblocked.
VFS: mounting devfs at /dev
VFS: mounting procfs at /proc
program zpool.so returned 1
BSD shrinker: event handler list found: 0xffffa00001d02580
	BSD shrinker found: 1
BSD shrinker: unlocked, running
[I/43 dhcp]: Waiting for IP...
[I/246 dhcp]: Server acknowledged IP for interface eth0
eth0: 192.168.122.89
[I/246 dhcp]: Configuring eth0: ip 192.168.122.89 subnet mask 255.255.255.0 gateway 192.168.122.1 MTU 1500
Running from /init/30-auto-00: /libhttpserver.so &!
could not load libicuuc.so.50
could not load libicui18n.so.50
could not load libicudata.so.50
Failed to load object: /tools/ifconfig.so. Powering off.

**Comment by nyh** [comment]
Perhaps instead of adding ifconfig.so to *every* image (like you did in your proposed patch), we should change the README to just ask people to do "scripts/build image=mgmt" before trying to run ifconfig.so?

The question is whether we really want ifconfig.so on every image. Does anyone really need or want this ifconfig.so, except mgmt's shell which assumes it's there?

**Comment by hpccdiaz** [comment]
I am getting same error when trying to execute the NFS test... any idea about how to solve it?

carlos@node0101:~/osv$ sudo scripts/run.py -e "modules/nfs-tests/tst-nfs.so" --verbose
OSv v0.24-108-gca6bcd4
4 CPUs detected
Firmware vendor: Bochs
bsd: initializing - done
VFS: mounting ramfs at /
VFS: mounting devfs at /dev
net: initializing - done
vga: Add VGA device instance
eth0: ethernet address: 52:54:00:12:34:56
virtio-blk: Add blk device instances 0 as vblk0, devsize=10737418240
random: virtio-rng registered as a source.
random: <Software, Yarrow> initialized
VFS: unmounting /dev
VFS: mounting zfs at /zfs
zfs: mounting osv/zfs from device /dev/vblk0.1
random: device unblocked.
VFS: mounting devfs at /dev
VFS: mounting procfs at /proc
program zpool.so returned 1
BSD shrinker: event handler list found: 0xffffa00001d0f280
	BSD shrinker found: 1
BSD shrinker: unlocked, running
[I/42 dhcp]: Waiting for IP...
[I/245 dhcp]: Server acknowledged IP for interface eth0
eth0: 192.168.122.15
[I/245 dhcp]: Configuring eth0: ip 192.168.122.15 subnet mask 255.255.255.0 gateway 192.168.122.1 MTU 1500
Running from /init/30-auto-00: /libhttpserver.so &!
could not load libicuuc.so.52
could not load libicui18n.so.52
Failed to load object: modules/nfs-tests/tst-nfs.so. Powering off.

**Comment by nyh** [comment]
@hpccdiaz I don't think there's a bug here, the following works:
```
make image=tests nfs=true
scripts/run.py -e /tst-nfs.so
```
If you still think there's a bug here, please open a new issue and don't tack it onto an unrelated issue. Thanks.
